Orm/Xtensive.Orm/Orm/Domain.cs

```diff
@@ -44,6 +45,8 @@ public sealed class Domain : IDisposable, IHasExtensions
     private bool isDisposed;
     private Session singleConnectionOwner;
 
+    private bool IsDebugEventLoggingEnabled { get; set; }
+

```

Comment:
Do we need a private property that never changes (outside constructor, of course)? I assume ```private readonly``` field would be better for class-wide read-only usage.

Reply:
I've converted this property to readonly field